src/ports/postgres/modules/lda/test/lda.sql_in

```diff
@@ -448,12 +456,15 @@ SELECT lda_train(
     'lda_output_data',       -- output_data_table
     20,                      -- voc_size
     5,                       -- topic_num
-    3,                       -- iter_num
+    10,                       -- iter_num
     10,                      -- alpha
     0.01,                    -- beta
     2,                       -- evaluate_every
-    10                       -- perplexity_tol
+    100                      -- perplexity_tol
     );
 
 
-select assert(abs(perplexity[2] - perplexity[1]) >  .1, 'Perplexity toll is less than the perplexity difference') from lda_model ;
\ No newline at end of file
+select assert(abs(perplexity[2] - perplexity[1]) <10, 'Perplexity tol is less than the perplexity difference') from lda_model ;
```

Comment:
Why are we checking for `<10` if the tol is `100` ?

Reply:
I think can add another assert to all the dev-check tests `assert that all the perplexity values are unique`. What do you think ?

Reply:
do you mean if the length of the calculated perplexity values matches the distinct perplexity values? 
fixed other issues.

Reply:
No I mean adding an assert to check that all the perplexity values are different

Reply:
Added test case for distinct perplexity values as discussed.